players/migrations/0003_auto_20160715_0914.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+import players.models
+
+
+def move_data(apps, schema_editor):
+    Player = apps.get_model("players", "Player")
+    Avatar = apps.get_model("players", "Avatar")
+    Game = apps.get_model("players", "Game")
+
+    main_game = Game(pk=1, name="main")
```

Comment:
We do not want any data addition in our migrations. We have this mistake in rapid router, I'd like to avoid having the same issue here.

i.e. If someone starts with a blank project, without existing data, and runs the migrations as part of normal setup, it should not result in there being any `Game` objects.

We could do something conditional, e.g. if there are any avatars, make a default game for them, otherwise do nothing...